app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -91,6 +91,17 @@ def link_to_field(name, value, label = nil, facet_hash = {})
       link_to(label, main_app.search_catalog_path(state))
     end
 
+    def index_facet_link options={}
```

Comment:
I think this duplicates the functinality of `link_to_search: solr_name("keyword", :facetable)`